charts/sda-svc/values.yaml

```diff
@@ -491,7 +493,7 @@ s3Inbox:
 sftpInbox:
   name: sftpInbox
   repository: ghcr.io/neicnordic/sda-inbox-sftp
-  imageTag: v1.7.63
+  imageTag: v1.10.4
```

Comment:
this might cause some issues as it requires a new APIs see https://github.com/neicnordic/sda-inbox-sftp/pull/287/files ... if you use this version change EGA users API or revert to `1.9.3`

Reply:
Seems we need to make changes to the fake cega server to go past 1.9.3, so it's better to do that as a separate PR.
On the same note, we should add a test that actually verifies that a user can upload a file in the inboxes.

Reply:
true, that also sounds like another PR.